hydra-node/bench/tx-cost/TxCost.hs

```diff
@@ -140,14 +139,15 @@ computeCollectComCost =
     let spendableUTxO = getKnownUTxO stInitialized
     pure (fold committedUTxOs, unsafeCollect cctx headId (ctxHeadParameters ctx) utxoToCollect spendableUTxO, getKnownUTxO stInitialized <> getKnownUTxO cctx)
 
-computeDecommitCost :: IO [(NumParties, TxSize, MemUnit, CpuUnit, Coin)]
-computeDecommitCost = do
+computeDecrementCost :: IO [(NumParties, TxSize, MemUnit, CpuUnit, Coin)]
+computeDecrementCost = do
   interesting <- catMaybes <$> mapM compute [1, 2, 3, 5, 10]
   limit <- maybeToList . getFirst <$> foldMapM (fmap First . compute) [50, 49 .. 11]
   pure $ interesting <> limit
  where
   compute numParties = do
-    (ctx, st, tx) <- generate $ genDecrementTx numParties
+    -- TODO: add decrementedOutputs to the result
```

Comment:
is this TODO still valid?

Reply:
Yes. I have not done it and this is a bit of procrastination.